app/views/services/new.html.erb

```diff
@@ -1,24 +1,24 @@
 <fieldset class="govuk-fieldset">
   <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
-    <h1 class="govuk-fieldset__heading">Add a Service</h1>
+    <h1 class="govuk-fieldset__heading"><%= t 'services.title' %></h1>
```

Comment:
I think `title` should refer to the browser page title, this is more of a heading?